pcs_detection/scripts/data/weights/using_negatives_fcn8_GREY_19_10_10_101037/config.json

```diff
@@ -0,0 +1,60 @@
+{
+    "AUGMENTATIONS": {
+        "number_aug": 100,
+        "horizontal_flip": true,
+        "brightness_range": null,
+        "rotation_range": 45,
+        "vertical_flip": true,
+        "zoom_range": 0.2
+    },
+    "LABEL_THICKNESS": 5,
+    "MODE": "VALIDATE",
+    "BACKGROUND_REDUCTION": 30,
+    "WEIGHT_ID": "using_negatives",
+    "PRE_PROCESS": {
+        "rgb": [
+            182.11864,
+            155.26283
+        ],
+        "edge": [
+            0,
+            3
+        ],
+        "grey": [
+            117
+        ]
+    },
+    "MIN_PIXELS_IN_IMG": 100,
+    "LEARNING_RATE": {
+        "reduce_patience": 1,
+        "start": 0.008,
+        "end": 0.0001,
+        "reduce_factor": 0.6
+    },
+    "N_EPOCHS": 100,
+    "IMG_DIMS": [
+        256,
+        256
+    ],
+    "USE_FULL_IMAGE": false,
+    "VAL_WEIGHT_PATH": "38.h5",
+    "TRAINING_DIRS": [
```

Comment:
Config is for infererence isn't it? Then it shouldn't need the training data, validation_data, learning rate, or background_reduction.

Reply:
That's true, none of those parameters are needed to run. I typically save off everything the weights were trained on as a reference for future training.

Reply:
Yeah. Maybe you could dump that into a log file or something. Or even store it in the h5 file. It just seems a bit confusing what are actual inputs.

Reply:
That makes sense, I'll put it in a log file

Reply:
Two files are now saved off after training. One is the full training config which is used for the example_weights.py since data_loader.py requires most parameters for label reconstruction. I also made an inference_config that has only requirements for inference